tests/features/syntax_highlight_test.py

```diff
@@ -160,3 +161,11 @@ def test_syntax_highlighting_tabs_after_line_creation(run, tmpdir):
 def test_does_not_crash_with_no_color_support(run):
     with run(term='xterm-mono') as h, and_exit(h):
         pass
+
+
+def test_file_without_extension(tmpdir):
+    f = tmpdir.join('f')
+    f.write('#!/usr/bin/env python3')
+    f.chmod(0o777)
+    tags = tags_from_path(str(f))
+    assert 'python' in tags
```

Comment:
this doesn't test any of the code from `babi`

also I commented previously about the `777` -- please follow that PR comment!